web/src/assets/locales/en.json

```diff
@@ -82,29 +82,29 @@
         },
         "allow_pr": {
           "allow": "Allow Pull Requests",
-          "desc": "Pipelines can run on pull requests."
+          "desc": "Allows pipelines to run on pull request events."
```

Comment:
```suggestion
          "desc": "Allow pipelines to run on pull requess."
```

Reply:
I think both with "s" and without is valid here in terms of grammar. Depends a bit on how you construct the sentence in your mind. 

Fine with both and OK to modify the current state. There's a typo which needs to be fixed.

Reply:
Sure both fine, just use one way consistently :) Right now there is `Allow deployments ...` already used.